datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -216,6 +216,14 @@ def __init__(self, *args, **kwargs):
         logger = logging.getLogger('{}.{}'.format(__name__, self.name))
         self.log = CheckLoggingAdapter(logger, self)
 
+        exclude_metrics_filters = self.instance.get('exclude_metrics_filters', []) if instance else []
```

Comment:
Maybe remove by doing:

```python
    def _create_metrics_pattern(self, config, option_name, duplicates_to_check=None):
```

Reply:
That's a good idea, but then I'll have to re-fetch the exclude filters while checking for include/exclude overlap if that's fine with you

Reply:
Hmm, is that check useful?

Reply:
It doesn't actually do anything, just informs the user

Reply:
It was in the include/exclude labels logic, but probably isn't needed

Reply:
I removed the check since it doesn't provide much value and applied your suggestion 👍